graphql/resolvers/basicDashboardSentences.ts

```diff
@@ -0,0 +1,68 @@
+import { authOptions } from "@/pages/api/auth/[...nextauth]"
+import { type MyContext } from "@/pages/api/graphql"
+import { PgBasicSentence, PgQueryError, PgQueryResponse } from "@/types"
+import { pool } from "@/utils/postgres"
+import { getServerSession } from "next-auth"
+import { Arg, Ctx, Query, Resolver } from "type-graphql"
+import redis from "@/utils/redis"
+import { GetSentences, BasicSentence } from "../schemas/dashboard"
+
+@Resolver(GetSentences)
+export class BasicDashboardSentences {
+  @Query(() => GetSentences)
+  async basicDashboardSentences(
+    @Arg("runningAvg", () => String!) runningAvg: string,
+    @Ctx() { req, res }: MyContext
+  ): Promise<GetSentences> {
+    const {
+      user: { id: user },
+    } = await getServerSession(req, res, authOptions)
+
+    const args = [user, runningAvg]
+
+    // const cachedMetrics = await redis.get(`basic/${user}/${runningAvg}`)
+    // if (cachedMetrics)
+    //   return {
+    //     sentences: JSON.parse(cachedMetrics),
+    //   }
```

Comment:
I left this out because the caching system for this data will have to be completely redesigned. will do it in a separate ticket